### ISSUE TITLE
External links to open in new window/tab - Membership, Boost, Sponsors

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#450]
Many external links on the following pages open in the current tab/window but should open in a new tab/window:

* [ ] [Membership](https://www.jupiterbroadcasting.com/membership)
* [ ] [Sponsors](https://www.jupiterbroadcasting.com/sponsors)
* [ ] [Boost](https://www.jupiterbroadcasting.com/boost)
* [ ] Other? Should do an audit for other links on non-episode pages...

Anyone want to take this one??

**Comment by elreydetoda** [comment]
BTW, if you're new to markdown this is how you'd do it: https://stackoverflow.com/a/4425223

**Comment by gerbrent** [comment]
Wonderful @prestonw ! Please create a PR to our repo with these commits and we'll be happy to review/integrate them.

**Comment by TheHesster** [comment]
Wondering if I might be able to tackle this as a first ever contribution to a project. I don't believe a PR was every submitted by @prestonw

**Comment by TheHesster** [comment]
I haven't done a PR before either lol! Are you still planning to tackle this @prestonw 
If not I could give it a go and see if I can figure it out.

**Comment by CGBassPlayer** [comment]
I'd be happy to lend a hand on making a PR if needed! Just let me know

**Comment by TheHesster** [comment]
Sorry that I don't not know this answer. Do I fork this repo before cloning it down? This is hopefully going to be my first contribution to any open source project if I can figure it out. (Been reading Hugo docs, so I think I can figure it out)

**Comment by CGBassPlayer** [comment]
Yes! You would click the Fork Button and GitHub will create a fork to your account
![image](https://user-images.githubusercontent.com/34293941/215928551-7f412409-0990-4295-921d-7c0e7988077c.png)

Then you would `git clone https://github.com/TheHesster/jupiterbroadcasting.com.git`

Once you clone it down, I recommend using VSCode to edit this project, you can run a local instance with `make dev` as long as you have docker installed. If you do not want to use docker, make sure hugo is installed and I believe the command is `hugo serve`.  But the docker method is the preferred developer environment.

Don't forget to create a new branch from develop either! That is really useful if you want to keep a clean base copy of the code to branch other changes off from.

**Comment by TheHesster** [comment]
Thank you for the info @CGBassPlayer I've done exactly those steps after doing some research, I also set this repo as a remote named upstream based on my research too. Not sure if that's needed but oh well! Thanks again for the rundown.

**Comment by CGBassPlayer** [comment]
It can be helpful when pulling down the latest code straight to your machine. I usually will just use the Sync Fork on GitHub itself and pull down my `origin` remote.
![image](https://user-images.githubusercontent.com/34293941/215930434-024be53e-5aad-4706-81d1-306c0ff6333f.png)
Its just 2 different ways to accomplish the same thing

**Comment by TheHesster** [comment]
Ah I see. That makes sense too. Thanks

**Comment by TheHesster** [comment]
Submitted the PR. I thought I was filling out a PR originally to my own develop branch but I realized I was actually submitting the real PR. Fingers crossed.

**Comment by CGBassPlayer** [comment]
So if you want to do a PR within your fork, just make sure the base and compare branches are on the same repo.

This is what it looks like comparing your branch to the JB repo
![image](https://user-images.githubusercontent.com/34293941/215944203-c88c5ca6-d2f8-44d9-b612-54ef72a56417.png)

And this is what it looks like when you compare against your own branch
![image](https://user-images.githubusercontent.com/34293941/215944087-97d345ac-573c-4b87-9a09-630f2368a3e1.png)

Sometimes it will simplify to this view as well since it would be in the same repo
![image](https://user-images.githubusercontent.com/34293941/215944578-3c4cc210-32cf-4e21-b0cb-4e8c63ae5c03.png)


But I reviewed the PR and it looked ok in my testing. Someone with write access will have to be the one to merge it though.

**Comment by CGBassPlayer** [comment]
@gerbrent I am looking for a little clarification on this issue. Do you want the pages those pages listed to open in a new tab, or just the links on those pages?

**Comment by TheHesster** [comment]
Thanks @CGBassPlayer I started the pull request from within vscode so that's what confused me. I think I understand it now and where I went wrong. I think I just selected the wrong remote since I had set this one up as a remote as well as my repo.

The changes I added were to make all external links site wide open in new tabs by default. Using this page from Hugo documentation as reference. https://gohugo.io/templates/render-hooks/

If I was incorrect about what was wanted I'm sorry about that!
